realm/kotlin-extensions/src/androidTest/kotlin/io/realm/CoroutineTests.kt

```diff
@@ -28,19 +27,14 @@ class CoroutineTests {
     private lateinit var configuration: RealmConfiguration
     private lateinit var testDispatcher: TestCoroutineDispatcher
     private lateinit var testScope: TestCoroutineScope
-    private lateinit var realm: Realm
 
     @Before
     fun setUp() {
         testDispatcher = TestCoroutineDispatcher()
         testScope = TestCoroutineScope(testDispatcher)
         configuration = configFactory.createConfiguration()
-        realm = Realm.getInstance(configuration)
-    }
 
-    @After
-    fun tearDown() {
```

Comment:
Not having a tearDown would leave a realm instance open after the latest test execution, isn't it?

Reply:
No. I don't really need to use a global realm instance for these tests. Now instead I use Kotlin's `use` scope function, which receives the instance of the object you're calling it on (which has to be a `Closeable`) and calls `close()` automatically at the end of the closure.